Update g-suite-apps-audit-source.md (#1682)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source.md

## Configuring a G Suite Apps Audit Source

> Once you've set up a Hosted Collector and have your credentials ready, you're all set to configure the Source. Each G Suite App must have its own source. Follow these steps for each G Suite App you wish to collect.

Reviewer: ```suggestion
Once you've set up a Hosted Collector and have your credentials ready, you're all set to configure the Source. Each G Suite App must have its own source. For each G Suite App you wish to collect, follow the steps below.
```

To configure a G Suite Apps Audit Source:

> 1. In Sumo Logic select **Manage Data \> Collection > Collection**. 

Reviewer: ```suggestion
1. In Sumo Logic select **Manage Data** > **Collection** > **Collection**. 
```

1. In the Collectors page, click **Add Source** next to a Hosted Collector.
1. Select **G Suite Apps Audit**.
1. For **Name**, type the name of the Source. Enter an optional Description.

> 1. For **Application**, Select the G Suite app you wish to collect using this source. Steps may be repeated for each G Suite app you want to collect from.

Reviewer: ```suggestion
1. For **Application**, select the G Suite app that you’d like this Source to collect data from. Steps may be repeated for each G Suite app you want to collect from.
```

> 1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory.`

Reviewer: ```suggestion
1. **Source Category**. Enter a string to tag the output collected from the source. The string that you supply will be saved in a metadata field called `_sourceCategory`.
```
